com.unity.render-pipelines.universal/Runtime/Passes/GBufferPass.cs

```diff
@@ -65,16 +63,12 @@ public override void Configure(CommandBuffer cmd, RenderTextureDescriptor camera
                 if (i == m_DeferredLights.GBufferNormalSmoothnessIndex && m_DeferredLights.HasNormalPrepass)
                     continue;
 
-                RenderTextureDescriptor gbufferSlice = cameraTextureDescriptor;
-                gbufferSlice.depthBufferBits = 0; // make sure no depth surface is actually created
-                gbufferSlice.stencilFormat = GraphicsFormat.None;
-                gbufferSlice.graphicsFormat = m_DeferredLights.GetGBufferFormat(i);
-                cmd.GetTemporaryRT(m_DeferredLights.GbufferAttachments[i].id, gbufferSlice);
```

Comment:
How are you allocating / deallocating gbuffer RTs?

Reply:
It is done in UniversalRenderer